common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -127,6 +127,8 @@ def get(self, key):
     def set(self, key, value):
         if key.scope == Scope.children:
             self._children = value
+        elif key.scope == Scope.parent:
```

Comment:
@cpennington @sanfordstudent  Note: I needed to make this additional change in order to fix some failing unit tests in CSM.